spa/src/components/account/ResetPassword/ResetPasswordForm.js

```diff
@@ -25,12 +23,12 @@ function ResetPasswordForm({ onResetPasswordSubmit, loading }) {
   };
 
   const password = watch('password', '');
-  const password1 = watch('password1', '');
-  const disabled = password1 === '' || password === '' || loading;
+  const confirmPassword = watch('confirmPassword', '');
+  const disabled = !confirmPassword || !password || loading;
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <S.InputLabel hasError={errors.password}>Password</S.InputLabel>
+    <form onSubmit={disabled ? () => {} : handleSubmit(onSubmit)}>
```

Comment:
Same as before: if submit button is disabled, don't need this
```suggestion
    <form onSubmit={handleSubmit(onSubmit)}>
```

Reply:
[nrh-cklimas](https://github.com/nrh-cklimas) had this suggestion for a fallback. And it does not bother us, so lets keep i would say.